### PR TITLE
[new] Add vuikit@0.6.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by kytwb**
Reference issue https://github.com/cdnjs/cdnjs/issues/9308

# Profile of the lib
 * Git repository (required): https://github.com/vuikit/vuikit
 * Official website (optional, not the repository): https://vuikit.github.io/vuikit
 * NPM package url (optional): https://www.npmjs.com/package/vuikit
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 333
   - Count of watchers: 19
   - Count of forks: 24
 * NPM download stats (optional):
   - Downloads in the last day: 23
   - Downloads in the last week: 85
   - Downloads in the last month: 268

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@kytwb would you please also provide minified `vuikit.common.js`?

File: ajax/libs/vuikit/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "npmName": "vuikit",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "/vuikit.common.js"
```
Comment (PeterDaveHello) [inline ajax/libs/vuikit/package.json]: Please remove the leading slash, thanks.

@nolsherry let's add this part in fix tool and the test.

**Comment by kytwb** [comment]
@PeterDaveHello I'm checking out the repository on another computer, it's just impossible to work on with my old machine. I'll continue as soon as possible. Do you also want me to sqash the commits?

**Comment by PeterDaveHello** [comment]
@kytwb No worry, you can do it online, I can squash for you, and please make sure you enabled sparse-checkout and shallow clone to speed it up for 100x faster, thanks!

**Comment by kytwb** [comment]
@PeterDaveHello the minified version is already in the lib folder. It doesn't follow the `.min` naming convention. Minified version is `vuikit.js`. Map file `vuikit.js.map` also provided. Do you want me to specify the minified version in `filename`?

**Comment by PeterDaveHello** [comment]
@kytwb Okay, let's keep as what it looks like from upstream, thanks.

**Comment by PeterDaveHello** [comment]
@kytwb thanks for your contribution!

**Comment by miljan-aleksic** [comment]
Hei guys, thank you for this! I could definitely follow the naming convention, is actually how should it be. Let me know if I'm still on time for that change.

**Comment by PeterDaveHello** [comment]
@miljan-aleksic it's fine, no worry :)

**Comment by miljan-aleksic** [comment]
Thanks, @PeterDaveHello. If I do change it in some future, should I announce it with a new ticket?

**Comment by PeterDaveHello** [comment]
@miljan-aleksic Yes please

**Pull request merged**